test/integration/user_can_view_items_from_an_individual_artist_test.rb

```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class UserCanViewItemsFromAnIndividualArtistTest < ActionDispatch::IntegrationTest
```

Comment:
Also planning to ignore this hound issue, I don't think having a long test title is a problem.